tests/scripts/utils.sh

```diff
@@ -107,6 +107,12 @@ function run_suite() {
   fi
   
   SUITE_ID=$1 TEST_TAGS=$2 MODEL=$8 make test-e2e
+
+  if ! [ -z ${RESPONSE_EVALUATION:-} ]; then
```

Comment:
the intent is that we're going to have a wholly separate test job that sets this env variable and thereby runs the response evaluation check, right?

given that, do you actually want "make test-e2e" to also run, when you're doing that?  or should this be an either/or situation?  (either run the e2e tests, or run the response evaluation report/check)